packages/app/src/app/components/CreateNewSandbox/CreateSandbox/elements.ts

```diff
@@ -178,6 +168,7 @@ export const TemplateButton = styled.button`
   color: #e5e5e5;
   transition: background ${props => props.theme.speeds[2]} ease-out;
   outline: none;
+  max-width: 235px;
```

Comment:
any smarter ways of limiting the width of the grid children?

Reply:
Where do we need this for? I assume the grid children automatically scale based on the amount of grid columns?

Reply:
I fixed the grid setup, to ensure all columns are equal even if they overflow